pkg/commands/options/validate.go

```diff
@@ -26,11 +29,34 @@ In a future release this will be an error.
 -----------------------------------------------------------------
 `
 
+const platformsFlagWarning = `ERROR!
```

Comment:
I think we can do without the warning if we're returning an error.

Reply:
done, thanks for the review and feedback! PTAL